backend-api/src/functions/common/session/session.ts

```diff
@@ -27,6 +28,11 @@ export interface BaseSessionAttributes {
   redirectUri?: string;
 }
 
+export interface BiometricTokenSessionAttributes extends BaseSessionAttributes {
```

Comment:
```suggestion
export interface AuthSessionCreatedSessionAttributes extends BaseSessionAttributes {
```

^ matches the existing naming pattern, where we follow the name of the session state.